src/GraphsOptim.jl

```diff
@@ -7,12 +7,15 @@ module GraphsOptim
 
 using Graphs: AbstractGraph, is_directed
 using Graphs: vertices, edges, nv, ne, src, dst, inneighbors, outneighbors
+using Graphs: complement, maximal_cliques
```

Comment:
Would you be interested in coding an ILP-based `maximal_cliques` function for this package? That way we can be self-reliant

Reply:
I'm not sure that an ILP can return all possible maximal cliques.  Wouldn't it only give the (single) maximum clique?

Reply:
oh right